src/controllers/userController.js

```diff
@@ -73,6 +73,62 @@ const logIn = async (req, res) => {
   }
 };
 
+/******************************************************
+ * @GETUSER
+ * @route /api/v1/login
+ * @method GET
+ * @description retrieve user data from mongoDb if user is valid(jwt auth)
+ * @returns User Object
+ ******************************************************/
+
+const getUser = async (req, res) => {
+  const { userId } = req.user;
+  try {
+    const user = await userModel.findById(userId);
+    return res.status(200).json({
+      success: true,
+      message: "User data got  sucessfully",
+      data: req.user,
+    });
+  } catch (error) {
+    return res.status(400).json({
+      success: false,
+      message: error.message,
+    });
+  }
+};
+/******************************************************
+ * @GETADMIN
+ * @route /api/v1/admin
+ * @method GET
+ * @description retrieve user data from mongoDb if user is valid(jwt auth)
+ * @returns User Object
+ ******************************************************/
+
+const getAdmin = async (req, res) => {
```

Comment:
I would change this controller name so that it reflects its purpose. Here's an alternative: `grantAdminRole`.